### PR TITLE
MEB-48: add Forums to contact page

### PR DESCRIPTION
**Pull request by caroline-gschwend**
Feel free to suggest different wording for this section.

### CODE REVIEW

File: metabrainz/templates/index/contact.html [hunk 1]
```diff
@@ -63,6 +63,12 @@ <h1 class="page-title">Contact</h1>
       <strong><a href="mailto:support@metabrainz.org">support@metabrainz.org</a></strong>
     </p>
   </div>
+  
+  <h3>Forums</h3>
+  <p>
+    Please come ask your questions at our<a href="https://community.metabrainz.org/">Forums</a>! We would love to
+    answer any questions you may have.
```
Comment (mayhem) [inline metabrainz/templates/index/contact.html]: Needs a space after our, otherwise you get ourForums.

Comment (gentlecat) [inline metabrainz/templates/index/contact.html]: Also I don't think `Forums` needs to be capitalized.

**Comment by caroline-gschwend** [comment]
Thank you @mayhem @gentlecat

**Comment by gentlecat** [comment]
Thanks. :+1:

**Pull request merged**